docs/.snippets/code/build-on-polkadot/xcm/asset-transfer-api/reference/transfer-arg-opts.md

```diff
@@ -0,0 +1,104 @@
+Options for customizing the claim assets transaction. These options allow you to specify the transaction format, fee payment details, weight limits, XCM versions, and more.
+
+??? child "Show more"
+
+    `format` ++"T extends Format"++ 
+        
+    Specifies the format for returning a transaction.
+
+    ??? child "Type `Format`"
+
+        ```ts
+        export type Format = 'payload' | 'call' | 'submittable';
+        ```
+    
+    ---
+
+    `paysWithFeeOrigin` ++"string"++
+    
+    The Asset ID to pay fees on the current common good parachain. The defaults are as follows:
+
+        - Polkadot AssetHub - DOT
+        - Kusama AssetHub - KSM
```

Comment:
```suggestion
    The Asset ID to pay fees on the current common good parachain. The defaults are as follows:

    - Polkadot Asset Hub - `'DOT'`
    - Kusama Asset Hub - `'KSM'`
```

If these are the string values, we should represent them as such. I also update the indentation because as it was indented it was rendering as a list in a code snippet, now it just renders as a list - weird framework stuff haha 😅